inji-verify/src/components/commons/CheckInternetConnectivity.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+
+const makeAnApiCall = async(): Promise<boolean> => {
+    let response;
+    try {
+        // Try making an api call if the window.navigator.onLine is true
+        response = await fetch('https://api.github.com', {
```

Comment:
is this the standard way of doing? what if github is down which happens usually?